src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp

```diff
@@ -1112,7 +1112,7 @@ namespace energyplus {
           auto& zoneVentilation = m_idfObjects.emplace_back(IddObjectType::ZoneVentilation_DesignFlowRate);
           zoneVentilation.setName(tzName + " Ventilation per Floor Area");
           zoneVentilation.setString(ZoneVentilation_DesignFlowRateFields::ZoneorZoneListorSpaceorSpaceListName, tzName);
-          zoneVentilation.setString(ZoneVentilation_DesignFlowRateFields::ScheduleName, this->alwaysOnSchedule().nameString());
+          zoneVentilation.setString(ZoneVentilation_DesignFlowRateFields::ScheduleName, modelObject.model().alwaysOnDiscreteSchedule().nameString());
```

Comment:
So windows was failing due to the use of that `auto& zoneVentilation = m_idfObjects.emplace_back(...)`.

the call to `this->alwaysOnSchedule()` would find the `m_alwaysOnSchedule` member not initialized, so it would do `m_idfObjects.push_back`.... -> `m_idfObjects` was resized, and the `&zoneVentilation` reference was invalidated.